TrafficCrash/Pages/Index.cshtml.cs

```diff
@@ -27,15 +37,19 @@ public JsonResult OnGet()
             List<Vehicle_People_Crash_details> vehicle_n_people = new List<Vehicle_People_Crash_details>();
             using (WebClient webClient = new WebClient())
             {
-                string vehicleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/68nd-jvt3.json");
+     //           string vehicleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/68nd-jvt3.json");
+                string vehicleJson = GetData("https://data.cityofchicago.org/resource/68nd-jvt3.json");
                 TrafficCrashVehicle[] allVehicles = TrafficCrashVehicle.FromJson(vehicleJson);
 
-                string peopleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/u6pd-qa9d.json");
+                //                string peopleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/u6pd-qa9d.json");
+                string peopleJson = GetData("https://data.cityofchicago.org/resource/u6pd-qa9d.json");
+
                 TrafficCrashPeople[] allPeople = TrafficCrashPeople.FromJson(peopleJson);
-             
+
+                IDictionary<string, TrafficCrashVehicle> trafficCrashPeople = new Dictionary<string, TrafficCrashVehicle>();
```

Comment:
This looks like a shadow condition... you're declaring a variable on this line, trafficCrashPeople.  That variable is already declared as the iteration variable on the for-each loop.  So, you have two variables with the same name, which will likely lead to results other than what you expect.